src/main/java/Duke.java

```diff
@@ -1,10 +1,118 @@
+                addTask(tasks, newEvent);
+            } else {
+                printRetryMessage();
+            }
+            printLineBorder();
+            userInput = in.nextLine();
+            userCommand = userInput.split(" ")[0];
+        }
+    }
+
+    private static void printRetryMessage() {
+        System.out.println("Ohhh my sire please give me a valid command");
+    }
+
+    private static void printLineBorder() {
+        System.out.println("\t~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+    }
+
+    private static Event getEventFromInput(String userInput) {
+        String[] eventComponents = userInput.split("event | /at ");
+        String eventName = eventComponents[1];
+        String eventDetails = eventComponents[2];
+        Event newEvent = new Event(eventName, eventDetails);
+        return newEvent;
+    }
+
+    private static Deadline getDeadlineFromInput(String userInput) {
+        String[] deadlineComponents = userInput.split("deadline | /by ");
+        String taskName = deadlineComponents[1];
+        String taskDeadline = deadlineComponents[2];
+        Deadline newDeadline = new Deadline(taskName, taskDeadline);
+        return newDeadline;
+    }
+
+    private static Todo getTodoFromInput(String userInput) {
+        String taskName = userInput.split(" ", 2)[1];
+        Todo newTodo = new Todo(taskName);
+        return newTodo;
+    }
+
+    private static void resolveTask(ArrayList<Task> tasks, String userInput) {
+        int taskNumber = Integer.parseInt(userInput.split(" ")[1]);
+        tasks.get(taskNumber - 1).markAsDone();
+        System.out.println("\t> Excellent. This task shall be marked as done: \n\t  "
+                + tasks.get(taskNumber - 1).toString());
+    }
+
+    private static void listTasks(ArrayList<Task> tasks) {
+        System.out.println("\t> Here are your tasks, sire: ");
+        for (int i = 0; i< tasks.size(); i++) {
+            System.out.println("\t" + (i+1) + "." + tasks.get(i).toString());
+        }
+    }
+
+    private static void printGoodbye() {
+        printLineBorder();
+        System.out.println("\t> Goodbye, sire. Fare thee well.");
+        printLineBorder();
+    }
+
+    private static void printGreetings() {
+        printLineBorder();
+        System.out.println("\t> Good day, sire, Percival at your disposal.");
+        System.out.println("\t> How may I be of service?");
+        printLineBorder();
+    }
+
+    private static void addTask(ArrayList<Task> tasks, Task task) {
+        tasks.add(task);
+        printAddedTaskMessage(task);
+        printTaskCountMessage(tasks);
+    }
+
+    private static void printTaskCountMessage(ArrayList<Task> tasks) {
+        if (tasks.size() == 1) {
+            System.out.println("\t> Now you have " + tasks.size() + " task in the list.");
+        } else {
+            System.out.println("\t> Now you have " + tasks.size() + " tasks in the list.");
+        }
```

Comment:
I think it is possible to simplify this even further by applying the condition to whether to append an "s" or a "\0" following the task word and reducing it to a single line of code, but that's just my preference!